distribution/test/thirdparty/restito/pom.xml

```diff
@@ -28,7 +28,7 @@
         <dependency>
             <groupId>com.xebialabs.restito</groupId>
             <artifactId>restito</artifactId>
-            <version>0.7</version>
+            <version>0.8.2</version>
```

Comment:
Maybe make this a property in the `ddf.test/test` pom or even use dependency management?